client/src/components/puzzleShow.js

```diff
@@ -1,7 +1,115 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom"
+import ErrorList from "./ErrorList.js"
+import translateServerErrors from "../services/translateServerErrors.js"
+import BoxTile from "./BoxTile"
 
 const puzzleShow = (props) => {
```

Comment:
`PuzzleShow`

Reply:
You might be IMPORTING this file with a titelized PuzzlePage, but update the file name as well! 

It might be that Git is not recognizing that you are doing this, so use `git mv` 
https://stackoverflow.com/questions/11183788/in-a-git-repository-how-to-properly-rename-a-directory
```
git mv client/src/components/puzzleShow.js client/src/components/PuzzleShow.js
```